sources/packages/web/src/types/contracts/StudentContract.ts

```diff
@@ -133,3 +133,46 @@ export interface StudentDetail {
   pdStatus: string;
   hasRestriction: boolean;
 }
+
+/**
+ * Enumeration for Document purpose these are
+ * the dropdown options from the form student
+ * uploader, for the dropdown down field
+ * `Document purpose`.
+ *
+ */
+export enum DocumentPurpose {
```

Comment:
Is there a need to have these values hardcode or to these values be known in advance?
IMO these values could be freely managed by the form.io definition and saved to the DB as a plain string. In the future, Ministry can add more items to this dropdown.